db/migrations/2023_05_21_000000_create_api_permissions.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Migrations;
+
+use Engelsystem\Database\Migration\Migration;
+
+class CreateApiPermissions extends Migration
+{
+    /**
+     * Run the migration
+     */
+    public function up(): void
+    {
+        $db = $this->schema->getConnection();
+        $db->table('privileges')->insert([
+            ['name' => 'api', 'description' => 'Use the API'],
+        ]);
+        $db->table('groups')->insert([
+            ['id' => 40, 'name' => 'API'],
+        ]);
+
+        $shiftCoordinatorGroup = 60;
+        $apiId = $db->table('privileges')->where('name', 'api')->first()->id;
+        $db->table('group_privileges')->insert([
+            ['group_id' => $shiftCoordinatorGroup, 'privilege_id' => $apiId],
```

Comment:
I think it would be better to put this privilege into the buerocrat group for now